custom_components/tuya_local/devices/deltaco_shlw5m_ledstrip.yaml

```diff
@@ -0,0 +1,43 @@
+name: LED Strip WIFI
```

Comment:
WIFI is superfluous in the device name. And following HA naming convention, strip should be all lower case.

Reply:
Ok, removed WIFI anc changed strip to lower case.